src/test/java/ai/elimu/web/content/word/WordCreateControllerTest.java

```diff
@@ -54,4 +58,45 @@ public void testHandleSubmit_emptyText() throws Exception {
 //        assertEquals(HttpStatus.BAD_REQUEST.value(), mvcResult.getResponse().getStatus());
 //        assertEquals("content/word/create", mvcResult.getModelAndView().getViewName());
     }
+
+    @Test
+    public void testValidateDigitsInWord() throws Exception {
+        assertTrue(containGivenErrorCode(getBindingResult(getRequestWithSpecificText("10")).getAllErrors(), "WordNumbers"));
+        assertTrue(containGivenErrorCode(getBindingResult(getRequestWithSpecificText("'10'")).getAllErrors(), "WordNumbers"));
+        assertTrue(containGivenErrorCode(getBindingResult(getRequestWithSpecificText("\\\"10\\\"")).getAllErrors(), "WordNumbers"));
+        assertTrue(containGivenErrorCode(getBindingResult(getRequestWithSpecificText("Test10Test")).getAllErrors(), "WordNumbers"));
```

Comment:
@tomaszsmy "Test10Test" should be allowed, because we might have some cases like this, for example "2nd" and "3rd" being categorized as words.